package.json

```diff
@@ -281,7 +281,28 @@
                 "title": "Debug",
                 "icon": "$(bug)",
                 "category": "Jupyter",
-                "enablement": "notebookKernelCount > 0 && !jupyter.notebookeditor.debuggingInProgress && config.jupyter.experimental.debugging"
+                "enablement": "notebookKernelCount > 0 && !jupyter.notebookeditor.debuggingInProgress && !jupyter.notebookeditor.runByLineInProgress && config.jupyter.experimental.debugging"
```

Comment:
Could the 'debuggingInProgress' context key be true when run by line is in progress? So that the enablement clause here can be shorter?

Reply:
It can't, by we don't want users to be able to press RBL while debugging and vice versa.
Also we don't want them to be able to press RBL if they're already RBLing.